modules/release-notes/partials/docs-server-7.2.4-release-note.adoc

```diff
@@ -0,0 +1,104 @@
+
+[#release-724]
+== Release 7.2.4 (January 2024)
+
+Couchbase Server 7.2.4 was released in November 2023.
+This maintenance release contains new features and fixes to issues.
+
+[#new-features]
+=== New Features and Enhancements
+
+* REST interface added to set the retention period of audit files
+* Erlang upgraded to version{nbsp}25.
+* Added `Memcached` alert.
+
+
+=== Cluster Manager
+
+[#table-new-features-724-cluster-manager, cols="10,40"]
+|===
+|Feature | Description
+
+| https://issues.couchbase.com/browse/MB-59503/[MB-59503]
```

Comment:
@RayOffiah I'm asking @ggray-cb if he has more on this feature, which was backported from 7.6.

Reply:
Okay, let me know.  👍🏾